extensions/common/proto_util.cc

```diff
@@ -105,6 +118,7 @@ flatbuffers::DetachedBuffer extractNodeFlatBufferFromStruct(
   node.add_labels(labels_offset);
   node.add_platform_metadata(platform_metadata_offset);
   node.add_app_containers(app_containers_offset);
+  node.add_instance_ips(ip_addrs_offset);
```

Comment:
ip_addrs_offset is not initialized when size() == 0.

Reply:
This needs addition to unit tests I think.

Reply:
Are the same changes needed for `app_containers_offset` ?

Reply:
Not sure. Library might already be handling uninitialized offsets. Adding a test would make it clear.